src/marqo/errors.py

```diff
@@ -219,4 +219,4 @@ class ConfigurationError(InternalError):
 class DiskWatermarkBreachError(InternalError):
     """Error when OpenSearch disk watermark is breached"""
     code = "disk_watermark_breach_error"
-    status_code = HTTPStatus.INSUFFICIENT_STORAGE
+    status_code = HTTPStatus.TOO_MANY_REQUESTS
```

Comment:
Why are returning Too Many Requests status for storage though?

Reply:
@farshidz we need to return some 4xx error. By principal of "if two solutions are equal, choose the existing solution" I suggested 429 rather than 400, as this is what we were returning before, and what OpenSearch returns

Reply:
Will change this to a 400 error